models.py

```diff
@@ -28,4 +29,4 @@ def validate_rating(self, key, value):
         return value
 
     def __str__(self):
-        return self.restaurant.name + " (" + self.review_date.strftime("%x") +")"
+        return f"{self.restaurant.name} ({self.review.date:%x})"
```

Comment:
Generally f-strings are preferred when the expressions inside are fairly simple, since they're more readable and less error prone.